404.html

```diff
@@ -10,18 +10,18 @@
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>Documentation | MobiledgeX Developers</title>
-<meta name="description" content="The MobiledgeX Platform enables developers to dynamically scale and deploy applications on Telco data centers (cloudlets) geographically close to your end-users. Deploying to MobiledgeX&#039;s cloudlets provides applications the advantage of low latency, which can be extremely useful for real-time applications such as Augme..." />
+  <title>Documentation | EdgeXR Developers</title>
+<meta name="description" content="The EdgeXR Platform enables developers to dynamically scale and deploy applications on Telco data centers (cloudlets) geographically close to your end-users. Deploying to EdgeXR&#039;s cloudlets provides applications the advantage of low latency, which can be extremely useful for real-time applications such as Augme..." />
 <meta property="og:type" content="website" />
 <meta property="og:title" content="Documentation" />
-<meta property="og:description" content="The MobiledgeX Platform enables developers to dynamically scale and deploy applications on Telco data centers (cloudlets) geographically close to your end-users. Deploying to MobiledgeX&#039;s cloudlets provides applications the advantage of low latency, which can be extremely useful for real-time applications such as Augme..." />
+<meta property="og:description" content="The EdgeXR Platform enables developers to dynamically scale and deploy applications on Telco data centers (cloudlets) geographically close to your end-users. Deploying to EdgeXR&#039;s cloudlets provides applications the advantage of low latency, which can be extremely useful for real-time applications such as Augme..." />
 <meta property="og:url" content="http://dev-v3.test" />
-<meta property="og:site_name" content="MobiledgeX Developers" />
+<meta property="og:site_name" content="EdgeXR Developers" />
 <meta property="og:locale" content="en_US" />
 <meta name="twitter:card" content="summary_large_image" />
 <meta name="twitter:site" content="@mobiledgex" />
 <meta name="twitter:title" content="Documentation" />
-<meta name="twitter:description" content="The MobiledgeX Platform enables developers to dynamically scale and deploy applications on Telco data centers (cloudlets) geographically close to your end-users. Deploying to MobiledgeX&#039;s cloudlets provides applications the advantage of low latency, which can be extremely useful for real-time applications such as Augme..." />
+<meta name="twitter:description" content="The EdgeXR Platform enables developers to dynamically scale and deploy applications on Telco data centers (cloudlets) geographically close to your end-users. Deploying to EdgeXR&#039;s cloudlets provides applications the advantage of low latency, which can be extremely useful for real-time applications such as Augme..." />
```

Comment:
Yeah, there is a bunch of mobiledgex references leftover. For one - most of the pages have mobiledgex in the path. 
So these static sites are generated from the base markup, but only Matt knows now what was used to generate these sites, but the best way to address these references is through  a change in the base documentation. Also, @ThomasEXR was asking about "whitelabeling" the documentation - since it all now refers to `console.cloud.edgexr.org`.  I think for now, I'll remove the above reference to twitter just to get rid of this reference, but a lot of these should be addressed in a way that we wouldn't need to fix all of these manually again.

Reply:
I'm pretty sure the markup files were generated from something else. I think Matt used some website/service to build and host the website, and then used that to generate markup versions of everything to put into github. I can't remember what site/service he used, but I'm guessing the original sources are gone.